client/src/Component/views/KeyTerms/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
This is actually for ```client/src/Component/views/HowTo/inedx.style.js``` but there was nothing in it to comment on haha. The filed name accidentally got named "inedx" instead of "index. Just a little thing again, but we're very wary of all of our file names now! Haha